shell/platform/linux/fl_standard_message_codec_test.cc

```diff
@@ -0,0 +1,989 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/shell/platform/linux/public/flutter_linux/fl_standard_message_codec.h"
+#include "flutter/shell/platform/linux/testing/fl_test.h"
+#include "gtest/gtest.h"
+
+// FIXME(robert-ancell) Have two cases where endianess matters
+// FIXME(robert-ancell) Add tests for strings/lists/maps with more than 253
```

Comment:
Did you want to do this before landing? If not, it should be a TODO, as that's our normal comment style.